server/controllers/tripController.js

```diff
@@ -46,20 +45,64 @@ class TripController {
         DateOfTravel: trip.dataValues.dateOfTravel,
         DateOfReturn: req.body.returnDate,
       };
-      infoLogger(req, 200, message);
+
       util.setSuccess(200, message, data);
       return util.send(res);
+    
+    } catch(error){
```

Comment:
So how exactly are you testing these try-catch blocks? Because when I try to test them, I get issues in my code coverage (with Coveralls/nyc package).

Reply:
@fordarnold  I followed your advice on only logging internal `server error` from the *catch(error)* in the logs.